CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## vNext
 
 - Refactored environment variables.
+- Updated rpc url for cere blockchain.
```

Comment:
Use `Cere Network` instead of `cere blockchain`

Reply:
Updated.